horizontal_transport.md

```diff
@@ -0,0 +1,242 @@
+# Implementation of Horizontal Transport in bluejay (2-D Multicolumn Photochemical Model for Venus and Mars)
+
+## Overview
+
+Horizontal transport is crucial in accurately modeling atmospheric chemistry and dynamics on planets such as Venus and Mars. The updated code implements horizontal transport by combining advection (transport driven by horizontal winds) and diffusion (transport driven by concentration gradients). This document details the implementation approach, equations, and routines involved in achieving horizontal transport within our 2-D multicolumn photochemical model.
+
+## Key Components and Descriptions
+
+### 1. `speciesbclist_horiz`
+
+Defined in `MODEL_SETUP.jl`, this dictionary specifies boundary flux conditions for each chemical species at the domain edges. By default, both front and back edges have zero-flux conditions:
+
+```julia
+for sp in all_species
+    auto_speciesbclist_horiz[sp] = Dict("f"=>[[0.0 for ialt in 1:num_layers] for c in 1:2])
+end
+```
+
+This structure allows for easy modification to impose non-zero flux conditions:
+
+- Positive values represent influx into the domain.
+- Negative values represent outflux from the domain.
```

Comment:
Sign convention should probably associate positive with "toward higher index slices" rather than "into the atmosphere", since "into the atmosphere" is a different direction for the first and last slice.